src/Lithnet.GoogleApps/GmailServiceRequestFactory.cs

```diff
@@ -158,23 +199,23 @@ public void AddSendAs(string id, string sendAs)
         {
             id.ThrowIfNotEmailAddress();
 
-            GmailService service = this.GetService(id);
+            GmailService service = this.GetService(id, DefaultTimeout);
```

Comment:
Why don't we make the 'this.GetService(id)' overload use DefaultTimeout, rather than calling this overload everywhere. If we need to call with timeout.infinite when we can use the otherload that takes the TimeSpan parameter

Reply:
That would be good, but I decided against it when coding this, because I guess we will need to make all other methods in the RequestFactory also handle the TimeOut retry event?

I wanted to make as little changes as possible to the current code, to not ruin anything ;)